database/src/main/java/database/Repository.java

```diff
@@ -1,14 +1,8 @@
 package database;
 
-import database.exception.FailedDataBaseQueryException;
```

Comment:
after removing import FailedDataBaseQueryException class is not used anymore. delete it